src/services/ghost/strategies/AutoTriggerStrategy.ts

```diff
@@ -85,6 +85,7 @@ Provide non-intrusive completions after a typing pause. Be conservative and help
 		prompt += `Include surrounding text with the cursor marker to avoid conflicts with similar code elsewhere.\n`
 		prompt += "Complete only what the user appears to be typing.\n"
 		prompt += "Single line preferred, no new features.\n"
+		prompt += "NEVER suggest code that already exists in the file, including existing comments.\n"
```

Comment:
is this a good formulation? Maybe it should be something like just before/just after? I want autocomplete to duplicate quite often, and would even go as far as that is my main use of autocomplete.

A simple example would be `//kilocode change` should be suggested if it is in the file somewhere, and I start another comment